tests/unit/test_wheel.py

```diff
@@ -265,6 +266,19 @@ def test_get_entrypoints(tmpdir, console_scripts):
     )
 
 
+def test_assert_valid_entrypoint_ok():
+    _assert_valid_entrypoint("hello = hello:main")
+
+
+@pytest.mark.parametrize("entrypoint", [
+    "hello = hello",
+    "hello = hello:",
+])
+def test_assert_valid_entrypoint_fail(entrypoint):
+    with pytest.raises(MissingCallableSuffix):
+        _assert_valid_entrypoint(entrypoint)
```

Comment:
nit: maybe this should be named `_ensure_valid_entrypoint`? The `assert` in this context, makes me think there'd be an AssertionError.

Reply:
`_raise_for_invalid_entrypoint`? Ensure makes me think it will make it true, check makes me think it would return a bool.

Reply:
Makes sense to me!

Reply:
Addressed [here](https://github.com/pypa/pip/pull/6763/files#diff-79304b3f74b6bf37043da4d96ccd3755R319-R328).